test/functional/tools/ref_data_within_repeats.xml

```diff
@@ -0,0 +1,43 @@
+<tool id="ref_data_within_repeats" name="ref_data_within_repeats" version="1.0.0">
```

Comment:
I think you need to add this to test/samples_tool_conf.xml to ensure that test runs.